vividus-plugin-web-app/src/main/java/org/vividus/bdd/steps/ui/web/FieldSteps.java

```diff
@@ -104,6 +120,42 @@ public WebElement clearFieldLocatedByUsingKeyboard(Locator locator)
         return element;
     }
 
+    /**
+     * Gets a list of elements by <b>locator</b> and checks that it is
+     * sorted by text with using <b>sortingOrder</b> by element text.
+     * List should have more than 1 elements.
+     * @param locator locator to locate elements
+     * @param sortingOrder sorting order<br>
+     * <i>Available rule:</i>
+     * <ul>
+     * <li><b>ASCENDING</b> - will check that the list is sorted alphabetically,
+     * <li><b>DESCENDING</b> - will check that the list is sorted reverse alphabetically
+     * </ul>
+     */
+    @Then("elements located `$locator` are sorted by text in $sortingOrder order")
+    public void areElementSorted(Locator locator, SortingOrder sortingOrder)
+    {
+        List<WebElement> elements = baseValidations.assertIfAtLeastNumberOfElementsExist(
+                String.format("Fields by locator: %s", locator), locator, 2);
+        if (elements.size() >= 2)
```

Comment:
If the quantity <2 an empty list will be returned? Make sense to refactor to check if the list is not empty